frontend/src/scenes/max/maxContextLogic.ts

```diff
@@ -375,7 +368,7 @@ export const maxContextLogic = kea<maxContextLogicType>([
                     .map((item): MaxContextItem | null => {
                         switch (item.type) {
                             case MaxContextType.INSIGHT:
-                                return insightToMaxContext(item.data)
+                                return insightToMaxContext(item.data, item.filtersOverride, item.variablesOverride)
```

Comment:
Nitpick: `addOrUpdateContextInsight` calls `insightToMaxContext` (right below) with just one param. How does it work?

Reply:
The same for the `insightToMaxContext` calls in the `dashboardToMaxContext` 👀

Reply:
All good questions. I'll try to figure out exactly what's happening with all the branches and make sure nothing changes, then ask for another look.